services/parser-database/connector.py

```diff
@@ -20,20 +21,23 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
```

Comment:
Is there a test for this function already?

Reply:
Yes!